packages/zoe/src/contractSupport/zoeHelpers.js

```diff
@@ -305,7 +305,7 @@ const reverse = (keywordRecord = {}) => {
  * @param {ZCFSeat} fromSeat
  *   The seat in contractA to take the offer payments from.
  *
- * @param {ZCFSeat} [toSeat=fromSeat]
+ * @param {ZCFSeat} [toSeat]
```

Comment:
This one should probably stay, unless the function is updated to actually use an ES6 parameter default.
```patch
 export const offerTo = async (
   zcf,
   invitation,
   keywordMapping,
   proposal,
   fromSeat,
-  toSeat,
+  toSeat = fromSeat,
   offerArgs,
 ) => {
   if (keywordMapping === undefined) {
     keywordMapping = harden({});
   }
 
-  const definedToSeat = toSeat !== undefined ? toSeat : fromSeat;
-
   const zoe = zcf.getZoeService();
   const mappingReversed = reverse(keywordMapping);
 
   const newKeywords =
     proposal !== undefined
       ? mapKeywords(proposal.give, mappingReversed)
       : harden({});
 
   // the proposal is in the other contract's keywords, but we want to
   // use `proposal.give` to withdraw
   const payments = await withdrawFromSeat(zcf, fromSeat, newKeywords);
 
   // Map to the other contract's keywords
   const paymentsForOtherContract = mapKeywords(payments, keywordMapping);
 
   const userSeatPromise = E(zoe).offer(
     invitation,
     proposal,
     paymentsForOtherContract,
     offerArgs,
   );
 
   const depositedPromiseKit = makePromiseKit();
 
   const doDeposit = async payoutPayments => {
     // after getPayouts(), getFinalAllocation() resolves promptly.
     const amounts = await E(userSeatPromise).getFinalAllocation();
 
     // Map back to the original contract's keywords
     const mappedAmounts = mapKeywords(amounts, mappingReversed);
     const mappedPayments = mapKeywords(payoutPayments, mappingReversed);
-    await depositToSeat(zcf, definedToSeat, mappedAmounts, mappedPayments);
+    await depositToSeat(zcf, toSeat, mappedAmounts, mappedPayments);
     depositedPromiseKit.resolve(mappedAmounts);
   };
```